### PR TITLE
main/py-virtualenv: py3 support; modernize

### PR DESCRIPTION
**Pull request by TBK**

### CODE REVIEW

File: main/py-virtualenv/APKBUILD [hunk 1]
```diff
@@ -8,21 +9,50 @@ pkgdesc="Virtual Python Environment builder"
...
 
 package() {
-	cd "$builddir"
-	python2 setup.py install --prefix=/usr --root="$pkgdir" || return 1
+	mkdir -p "$pkgdir"
+}
+
+_py2() {
+  replaces="$pkgname"
+  depends="${depends//py-/py2-} py2-pip"
```
Comment (jirutka) [inline main/py-virtualenv/APKBUILD]: `depends` is empty, so `${depends//py-/py2-}` is useless here.

File: main/py-virtualenv/APKBUILD [hunk 2]
```diff
@@ -8,21 +9,50 @@ pkgdesc="Virtual Python Environment builder"
...
+  _py python2
+}
+ 
+_py3() {
+  depends="${depends//py-/py3-}" # pip is part of the python3 pkg
```
Comment (jirutka) [inline main/py-virtualenv/APKBUILD]: `depends` is empty, so this is useless here.

File: main/py-virtualenv/APKBUILD [hunk 3]
```diff
@@ -8,21 +9,50 @@ pkgdesc="Virtual Python Environment builder"
...
 build() {
 	cd "$builddir"
-	python2 setup.py build || return 1
+
+	python2 setup.py build
+	python3 setup.py build
+}
+
+check() {
+  cd "$builddir"
```
Comment (jirutka) [inline main/py-virtualenv/APKBUILD]: Please use only tabs for indentation (and alignment) in APKBUILDs and other shell scripts, not spaces. If you install http://editorconfig.org/ to your editor, it’ll take care of it for you.

File: main/py-virtualenv/APKBUILD [hunk 4]
```diff
@@ -8,21 +9,50 @@ pkgdesc="Virtual Python Environment builder"
 url="http://www.virtualenv.org/en/latest/"
 arch="noarch"
 license="MIT"
-depends="python2 py2-pip"
-makedepends="python2-dev"
+depends=""
+makedepends="python2-dev python3-dev" # add back py-tox for check()
```
Comment (jirutka) [inline main/py-virtualenv/APKBUILD]: What does `# add back py-tox for check()` mean? Dependencies for check are defined in `checkdepends`.

Comment (TBK) [reply main/py-virtualenv/APKBUILD]: I figured that out after looking throught the abuild code, could not find any docs providing that info.

I will fix it in a sec.

**Comment by TBK** [comment]
@jirutka I fixed the things that you mentioned :)

**Pull request closed without merging**